WordPress/Classes/ViewRelated/Post/Scheduling/CalendarCollectionView.swift

```diff
@@ -142,26 +213,93 @@ class DateCell: JTACDayCell {
             dateLabel.centerYAnchor.constraint(equalTo: centerYAnchor),
             dateLabel.centerXAnchor.constraint(equalTo: centerXAnchor)
         ])
+
+        leftPlaceholder.translatesAutoresizingMaskIntoConstraints = false
+        rightPlaceholder.translatesAutoresizingMaskIntoConstraints = false
+
+        addSubview(leftPlaceholder)
+        addSubview(rightPlaceholder)
+
+        NSLayoutConstraint.activate([
+            leftPlaceholder.widthAnchor.constraint(equalTo: widthAnchor, multiplier: 0.6),
+            leftPlaceholder.heightAnchor.constraint(equalTo: dateLabel.heightAnchor),
+            leftPlaceholder.rightAnchor.constraint(equalTo: centerXAnchor),
+            leftPlaceholder.centerYAnchor.constraint(equalTo: centerYAnchor)
+        ])
+
+        NSLayoutConstraint.activate([
+            rightPlaceholder.widthAnchor.constraint(equalTo: widthAnchor, multiplier: 0.5),
+            rightPlaceholder.heightAnchor.constraint(equalTo: dateLabel.heightAnchor),
+            rightPlaceholder.leftAnchor.constraint(equalTo: centerXAnchor, constant: 0),
+            rightPlaceholder.centerYAnchor.constraint(equalTo: centerYAnchor)
+        ])
+
+        bringSubviewToFront(dateLabel)
     }
 }
 
 extension DateCell {
-    func configure(with state: CellState) {
+    func configure(with state: CellState, hideInOutDates: Bool = false) {
```

Comment:
I'm not sure what `hideInOutDates` means. Could you add some documentation here?
```suggestion
    func configure(with state: CellState, hideInOutDates: Bool = false) {
```

Reply:
Added!